src/Core/Enums/RuleId.cs

```diff
@@ -140,30 +119,18 @@ public enum RuleId
 
         PatternsSupportedByControlType,
 
-        PatternsExpectedBasedOnParent,
-
-        PatternsExpectedBasedOnControlType,
-
-        HelpTextExists,
         HelpTextNotEqualToName,
 
         HyperlinkNameShouldBeUnique,
 
         IsControlElementPropertyExists,
-        IsControlElementPropertyCorrect,
 
-        // Axe.Windows.Rules
         IsContentElementPropertyExists,
         IsContentElementFalseOptional,
         IsContentElementTrueOptional,
         IsControlElementTrueOptional,
         IsControlElementTrueRequired,
 
-        // Scans
-        IsKeyboardFocusable,
```

Comment:
I searched in the repo for `IsKeyboardFocusable` and saw a few instances of a property by the same name, are these two separate things?

Reply:
Yes, they're different things. UIA exposes the `IsKeyboardFocusable` property, and the `IsKeyboardFocusable` scan was originally intended to check the validity of the property. The scan has since been split into individual rules such as `IsKeyboardFocusableShouldBeTrue`